src/bin/main.rs

```diff
@@ -568,6 +573,19 @@ impl GetOptsOptions {
             }
         }
 
+        if matches.opt_present("recursive") {
+            if let Some(skip) = options.skip_children {
+                if skip {
+                    return Err(format_err!(
+                        "Conflicting config options `skip_children` and `recursive` are \
+                        both enabled. `skip_children` has been deprecated and should be \
+                        removed from your config.",
```

Comment:
Not `config`, here the `skip_children` option should be removed from the command line

Reply:
😄 copy/paste strikes again! will update accordingly